app/models/like.rb

```diff
@@ -0,0 +1,10 @@
+class Like < ApplicationRecord
+  has_one :users
+  has_one :comments
+
```

Comment:
- [ ] Please update the associations as below because a like belongs to a user and a post.
 
 ```ruby
 belongs_to :author, class_name: 'User'
  belongs_to :post